community/index.md

```diff
@@ -5,9 +5,27 @@ sidebar:
   nav: community
 ---
 
-{% include toc %}
+## Engaging
 
+The Spinnaker OSS project was started at Netflix, and today thrives on the contributions and broad adoption of the public devops community. Here are some ways to get engaged - join us!
 
-## Contributors
+* [GitHub](https://github.com/spinnaker/spinnaker)
+* [Slack](http://join.spinnaker.io)
+* [StackOverflow](https://stackoverflow.com/search?q=spinnaker)
+
+## Core Contributors
+
+The list of contributors is growing with some core contributors below. Please reach out to us with a PR or on slack if you'd like to be added to this list!
+
+* [Cerner Corporation](http://cerner.com/)
+* [Google](https://opensource.google.com/)
+* [Kenzan](http://kenzan.com/)
+* [Microsoft/Azure Devops](https://github.com/azure-devops)
```

Comment:
That's the link they requested. I don't have a link for Target (target.com is the retail website, not a corporate site)